coverage-report/pom.xml

```diff
@@ -21,6 +21,27 @@
       https://github.com/jacoco/jacoco/wiki/MavenMultiModule
     -->
     <dependencies>
+        <dependency>
+            <groupId>org.dependencytrack</groupId>
+            <artifactId>alpine-common</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.dependencytrack</groupId>
+            <artifactId>alpine-model</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.dependencytrack</groupId>
+            <artifactId>alpine-infra</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.dependencytrack</groupId>
+            <artifactId>alpine-server</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Including these in our code coverage report significantly worsens our total coverage. Opted to include them anyway since we technically "own" this code now.

Reply:
So does it mean, unit tests need to be added alter for this part of code?

Reply:
Yeah ideally we'd improve coverage over time. 

But the total coverage being reduced for now will not negatively impact us really. Our PR checks only assert on coverage variation, and diff coverage (explanations [here](https://docs.codacy.com/faq/code-analysis/which-metrics-does-codacy-calculate/#code-coverage)). Not total coverage. 

So new code will continue to require good coverage, and PRs for it will pass as long as that is the case.